container/build.sh

```diff
@@ -116,10 +121,13 @@ main () {
   parse_parameters $@
 
   PROJECT_ROOT=$(git rev-parse --show-toplevel)
-  if [[ "$DISTRO" != "debian9" ]]; then
-    DIR="container/rbe-${DISTRO}"
+
+  if [[ "$TYPE" == "rbe-debian9" ]]; then
```

Comment:
lets have a mapping from the type to the directory so that we dont have to have these special cases